client/src/pages/AboutPage.jsx

```diff
@@ -1,12 +1,404 @@
+        pronouns: "(he/him)",
+        position: "Frontend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/razisyed-/",
+        website: "",
+        icon: RaziBoba,
+      },
+      {
+        name: "Darren Lee",
+        pronouns: "(he/him)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/darrenlee09/",
+        website: "",
+        icon: DarrenBoba,
+      },
+      {
+        name: "Lily Meng",
+        pronouns: "(she/her)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/lilyxmeng/",
+        website: "",
+        icon: LilyBoba,
+      },
+      {
+        name: "Anthony Camarillo",
+        pronouns: "(he/him)",
+        position: "Backend Software Engineer",
+        linkedIn: "https://www.linkedin.com/in/anthony-camarillo/",
+        website: "",
+        icon: AnthonyBoba,
+      },
+    ],
+  },
+};
+
+const mentors = [
+  {
+    name: "Brian Lee",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/brianscli/",
+    icon: MentorBoba,
+  },
+  {
+    name: "Michael Chan",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/michachan",
+    icon: MentorBoba,
+  },
+  {
+    name: "Joshua Wu",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://linkedin.com/in/wujoshua ",
+    websiteUrl: "https://www.joshuawu.me",
+    icon: MentorBoba,
+  },
+  {
+    name: "Emma Li",
+    pronouns: "(she/her)",
+    linkedInUrl: "https://www.linkedin.com/in/emmalili/",
+    icon: MentorBoba,
+  },
+  {
+    name: "Jordan Yep",
+    pronouns: "(he/him)",
+    linkedInUrl: "https://www.linkedin.com/in/jordanyep/ ",
+    icon: MentorBoba,
+  },
+];
 
 const AboutPage = () => {
+  const theme = useTheme();
+
   return (
     <BasePage>
-      <Typography mt={12}>About</Typography>
+      <Stack spacing={5} minWidth="100%" justifyContent="center">
+        <Typography
+          variant="pageTitle"
+          marginTop="6rem"
+          sx={{ width: "100%", borderBottom: "#e1d4c8 solid 1px" }}
+        >
+          About the Team
+        </Typography>
+        <Grid container direction="row">
+          <Grid item sm={12} md={6}>
+            <Typography variant="h6">Who We Are</Typography>
+            <Typography sx={{ fontSize: "14px" }}>
+              Welcome! This internship tracker was created by a group of student
+              volunteers and 5 mentors. We&apos;re college students from all
+              over the country who came together to build this certain feature
+              on the bobatalks website. We started this project in November 2022
+              of creating a fun and student-friendly way to look for and track
+              internships. We all hope this tool can help you be able to find
+              and track internships as you start going into application season
+              :) I also want to thank the original team, mentors, and all the
```

Comment:
Is this a message from someone?--being in the first person

Reply:
same comment as above related to the separate issue for the description